README.md

```diff
@@ -14,7 +14,10 @@ This SDK current supports the following versions of CloudEvents:
 
 Package **cloudevents** provides primitives to work with CloudEvents specification: https://github.com/cloudevents/spec.
 
-Sending CloudEvents:
+## Sending CloudEvents:
+
+Below we will provide samples on how to send cloudevents using the popular
+[`requests`](http://docs.python-requests.org/en/master/) library.
```

Comment:
- Remove language/branch from URL
- Use HTTPS to remove a 301 redirect

`https://docs.python-requests.org`